api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -131,6 +131,7 @@ void shouldProcessClaimFromPregnancyToChildTurningFour() throws JsonProcessingEx
         // invoke schedulers to process send email for card is about to be cancelled
         invokeAllSchedulers();
         assertThatCardIsAboutToBeCancelledEmailWasSent(claim);
+        // TODO assert that payment stopping email has been sent
```

Comment:
Ideally we should include the ticket number of the story where this gets implemented

Reply:
Updated